### PR TITLE
refactor TOC and samples page

### PR DESCRIPTION
**Pull request by laujan**

### CODE REVIEW

File: msteams-platform/tutorials/code-samples.md [hunk 1]
```diff
@@ -3,11 +3,21 @@ title: Microsoft Teams code samples
 description: Links and descriptions of sample applications for the Microsoft Teams developer platform
 keywords: Microsoft Teams developer samples
 ---
-# Code samples for the Microsoft Teams developer platform
+# Tutorials and code samples for the Microsoft Teams developer platform
 
-Here you will find a list of code samples that demonstrate various capabilities of the Microsoft Teams development platform and how to build apps to leverage those features.
+Here you will find a list of tutorials and code samples that demonstrate how you can extend the Teams developer platform capabilities by creating custom apps.
```
Comment (clearab) [inline msteams-platform/tutorials/code-samples.md:8]: Not a big fan of "developer platform" here - they aren't extending the developer platform, they are using the developer platform to extend Teams.

File: msteams-platform/tutorials/code-samples.md [hunk 2]
```diff
@@ -3,11 +3,21 @@ title: Microsoft Teams code samples
...
+
+| Capability| Learning Module|
+|--------|-------------|
+| Tabs  — embedded web experiences  |  [Create embedded web experiences with tabs for Microsoft Teams](https://docs.microsoft.com/learn/modules/embedded-web-experiences/) |
+| Webhooks and connectors  |  [Connect web services to Microsoft Teams with webhooks and Office 365 Connectors](https://docs.microsoft.com/learn/modules/msteams-webhooks-connectors/) |
+|Messaging extensions  | [Task-oriented interactions in Microsoft Teams with messaging extensions](https://docs.microsoft.com/learn/modules/msteams-messaging-extensions/)  |
+| Task modules |  [Collect input in Microsoft Teams with Task Modules](https://docs.microsoft.com/learn/modules/msteams-task-modules/) |
+| Conversational bots  | [Create interactive conversational bots for Microsoft Teams](https://docs.microsoft.com/learn/modules/msteams-conversation-bots/)  |
+
+## Getting started with code samples
```
Comment (clearab) [inline msteams-platform/tutorials/code-samples.md:20]: Should probably add links to the Python samples in the Messaging Extensions section.

Comment (clearab) [reply msteams-platform/tutorials/code-samples.md:20]: Also add a link to the all-up bot samples page in the intro of the bot section: https://github.com/Microsoft/BotBuilder-Samples

**Comment by laujan** [comment]
Hi @aosolis. This pull request addresses GitHub issue #1511. I believe the Hello World samples are too outdated to be helpful and I wanted to incorporate the Learn modules into our platform so I updated the samples page. Also, I wanted to separate the production-ready templates/integrations from the samples/tutorials so I refactored the TOC. I look forward to your feedback. Thank you!

File: msteams-platform/TOC.yml [hunk 3]
```diff
@@ -5,22 +5,22 @@
...
     href: tutorials/get-started-yeoman.md
-- name: Samples and templates
+  - name: Extend the Teams client with custom apps
+    href: tutorials/code-samples.md   
+- name: App templates and integrations
   items: 
   - name: Production-ready app templates
     href: samples/app-templates.md
-  - name: Production-ready Shifts Connectors
+  - name: Production-ready Shifts connectors
```
Comment (clearab) [inline msteams-platform/TOC.yml:22]: I believe "Connectors" here is a product name and should be capitalized, but I'm not 100%

**Comment by laujan** [comment]
Thanks, Andrew! Hope all is well with you and your family! I’m adjusting to the “new normal” but I miss the office comradery. 🙂

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by laujan** [comment]
closes #1511

**Pull request merged**